### ISSUE TITLE
Trees destroy village buildings

### ISSUE THREAD

**Issue by CrftTbls** [Darsenia/woodlands#3]
Title.
If a village is created in the biome generated by this mod, some of the houses will be destroyed.
Some of the walls have been replaced by leaves, and there are trees growing inside the buildings.

Used Mods:
bettervillage-forge-1.20.1-3.2.0
FarmersDelight-1.20.1-1.2.5
Oh-The-Trees-Youll-Grow-forge-1.20.1-1.3.1
tectonic-forge-1.20.1-2.4.1
TofuCraftReload-1.20.1-5.15.1.0
Woodlands_Extra-Universal-1.20.1-v1.3.1
Woodlands_Vanilla-Universal-1.20.1-v1.2.1

screenshot:
![2024-11-19_21 20 53](https://github.com/user-attachments/assets/6a314962-c248-411f-82f1-93b5bfd935c5)
![2024-11-19_21 21 08](https://github.com/user-attachments/assets/7fb41921-4221-4038-a36b-ce6891275b0f)
![2024-11-19_21 20 44](https://github.com/user-attachments/assets/b979d53c-c99d-47d7-a1b3-06f4b4713c06)

**Comment by Darsenia** [comment]
Yes, it's a known issue. This happenes because trees, generated by Oh The Trees You'll Grow mod do not consider any other structures in the world, including other trees and landscape objects. They do not check for them. Unfortunately, both Woodlands:Extra and The Core are simply datapack-based mods, that fully rely on this library mod in order to spawn custom trees, so I personally cannot fix this until CorgiTaco does it in their mod.

**Comment by Darsenia** [comment]
P.S. I'm not sure if I get it right from your modlist, but do not use both Woodlands mods at once, all the content from "The Core" is in the "Extra" mod.

**Comment by visualcycles** [comment]
Hey OP "Oh The Trees You'll Grow" was updated to take structures into account afaik so this shouldnt be an issue anymore
